capstone/src/main/webapp/youtubeTypescript.ts

```diff
@@ -37,14 +37,15 @@ const YOUTUBE_CATEGORIES = {
 };
 
 /** These variables will keep track of the data required for CSV output */
-let attributeData: string[][];
-let analyzedComments: string[];
+let ATTRIBUTE_DATA: string[][];
```

Comment:
I think this was added in another PR, but let's change these to no longer be global variables if possible

Reply:
addressed in "remove global variable dependence"